math-scala/src/main/scala/org/apache/mahout/math/algorithms/preprocessing/AsFactor.scala

```diff
@@ -28,10 +28,11 @@ import org.apache.mahout.math.drm.{drmBroadcast, _}
 import org.apache.mahout.math.drm.RLikeDrmOps._
 import org.apache.mahout.math.scalabindings.{dvec, _}
 import org.apache.mahout.math.scalabindings.RLikeOps._
+import sun.reflect.generics.reflectiveObjects.NotImplementedException
```

Comment:
my mistake - in my previous comment.. this should  be `UnsupportedOperationException`.